### DISCUSSION TITLE
Milyara kadar sayıların rakamları toplamı sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#173] [Soru Cevap]
1 den 1 milyara kadar olan sayıların tüm rakamlarının toplamını bulunuz.
Sayıların kendileri değil, rakamların toplamını istiyoruz.

**Comment by OmerB2** [comment]
```python
t = 0
for i in range(1,100000001):
    l = list(str(i))
    l = [int(s) for s in l]
    t = t+sum(l[0:-1])
print(t)
#1 milyar çok uzun sürdü ben de 1000,10000 ve 100000'le test ettim
```

**Reply by sonsuzus** [reply]
Daha hızlı yöntem ve algoritma deneyebilirsin.

**Reply by OmerB2** [reply]
nasıl hızlandırabilirim hocam

**Reply by sonsuzus** [reply]
Mesela 100 e kadar sayılarda kaç tane 1 kaç tane 2 kaç tane 9 vs var. Bu şekilde sayıları gruplara bölüp bakabilirsin.
